PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -64,7 +64,7 @@ def miniAOD_customizeCommon(process):
     process.selectedPatJets.cut = cms.string("pt > 10")
     process.selectedPatMuons.cut = cms.string("pt > 5 || isPFMuon || (pt > 3 && (isGlobalMuon || isStandAloneMuon || numberOfMatches > 0 || muonID('RPCMuLoose')))")
     process.selectedPatElectrons.cut = cms.string("")
-    process.selectedPatTaus.cut = cms.string("pt > 18. && tauID('decayModeFinding')> 0.5")
```

Comment:
It means that the hadronic tau has one of the allowed combinations of charged and neutral components - at the time of that release it had been either 1 track and 0/1/2 strips or 3 tracks. The PR by @andrewj314  allowed also other combinations with 2 tracks and strips in order to account for decreased efficiency that came with increased pileup. But anyway, it is almost 2 years old piece of sw, so it is probably outdated, so I suggest contacting tau conveners for more up-to-date info (I left CMS 1.5y ago ;)